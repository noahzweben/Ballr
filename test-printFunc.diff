1c1,5
<   ./test-printFunc failed on ./test-printFunc
---
> 5
> 10
> 15
> SDL could not initialize! SDL Error: No available video device
> Window initialization failed.
\ No newline at end of file
