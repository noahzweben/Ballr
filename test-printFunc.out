  ./test-printFunc failed on ./test-printFunc
