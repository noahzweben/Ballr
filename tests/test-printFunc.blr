func bool my_print(int a, int b, bool c){
	print(a);
	print(b);
	return c;
}

gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

		bool d = my_print(5, 10, true);
		if (d) {
			print(15);
		}
        
  	}
}