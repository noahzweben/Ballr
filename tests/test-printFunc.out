5
10
15
SDL could not initialize! SDL Error: No available video device
Window initialization failed.